app/src/debug/res/values/google_maps_api.xml

```diff
@@ -20,5 +20,5 @@
     Once you have your key (it starts with "AIza"), replace the "google_maps_key"
     string in this file.
     -->
-    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">YOUR_KEY_HERE</string>
+    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">AIzaSyDO1vgUcKL8TaGAnJoVNms3LIu11TkGcUw</string>
```

Comment:
so we are pushing the keys to master?

Reply:
i know some people had issues with their key and mike's key removed the problems so would this be a file that is constantly updated or could we keep a local version each and add it too the git ignore.

Reply:
The file is already on the git ignore,  we should not be pushing this information

Reply:
Removed it as well